browser/src/control/Control.NotebookbarWriter.js

```diff
@@ -2313,7 +2317,9 @@ L.Control.NotebookbarWriter = L.Control.Notebookbar.extend({
 							{
 								'type': 'toolitem',
 								'text': _UNO('.uno:FontworkGalleryFloater'),
-								'command': '.uno:FontworkGalleryFloater'
+								'command': '.uno:FontworkGalleryFloater',
+								// Fontwork export/import not supported in other formats.
+								'visible': isODF ? 'true' : 'false',
 							}
 						]
 					},
```

Comment:
to perfect result, it would be good to set VerticalText to bigtoolitem when there is no Fontwork so it will be consistent with other icons (single big button)

Reply:
Thanks, will add that in a followup.

Reply:
Vertical text as 'toolitem' with Fontwork item visible.
![image](https://user-images.githubusercontent.com/132960/211984175-a30d7148-d862-472a-ad06-e26d97cbdafe.png)

Vertical text as 'bigtoolitem' with Fontwork item hidden. The label is shown twice for some reason.
![image](https://user-images.githubusercontent.com/132960/211984382-c909a2dd-f473-4ae2-9ac2-adb26cf374a5.png)